src/style.css

```diff
@@ -1 +1,98 @@
-/* add styles here */
+/* stylelint-disable font-family-name-quotes */
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir';
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+body {
+  font-family: 'Avenir', sans-serif;
+  font-weight: 300;
+  margin-top: 20px;
+  height: 768px;
+}
+
+.icon {
+  min-width: 19px;
+  min-height: 19px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: 26px;
+  padding-left: 62px;
+}
+
+[data-qa='small'] {
+  background-position: 13px;
+  padding-left: 33px;
+  background-size: 0.6%;
+}
+
+.search {
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  transition: box-shadow;
+  font-family: 'Avenir', sans-serif;
+  font-weight: normal;
+  box-shadow: 0 1px 8px #3d4e6133;
+  width: 100%;
+  box-sizing: border-box;
+  font-size: 16px;
+}
+
+.search-2 {
+  margin-top: 20px;
+  height: 42px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  transition: box-shadow;
+  font-family: 'Avenir', sans-serif;
+  font-weight: normal;
+  box-shadow: 0 1px 8px #3d4e6133;
+  text-shadow: 0 4px 3px #3d4e6133;
+  width: 100%;
+  box-sizing: border-box;
+  font-size: 14px;
+}
```

Comment:
create some additional class and move there all common styles from every input and form, it should simplify the logic and support